genkeymap/readme.txt

```diff
@@ -1,43 +1,81 @@
-Creating a new keymap file.
----------------------------
+Keymap file description
+-----------------------
+
+The keymap files are used by the xrdp login screen, and also when
+sending keyboard input to a VNC server.
```

Comment:
```suggestion
sending keyboard input to a VNC server or xorgxrdp.
```
Or maybe a more general `backend`.

Reply:
The keymap files aren't used for xorgxrdp - only for the login screen and VNC.

For xorgxrdp, we try to ascertain the right arguments for XKB to set the keyboard up directly and pass those through. There's a different file involved (xrdp_keyboard.ini). It's all done here:-

https://github.com/neutrinolabs/xrdp/blob/cc7d5ef921938ff4a04e6efe5d0228712abaf471/libxrdp/xrdp_sec.c#L359-L597